test/language/statements/class/definition/fn-length-static-precedence.js

```diff
@@ -79,4 +75,3 @@ class E {
 }
 
 assert.sameValue(typeof E.length, 'function');
-assert(compareArray(Object.getOwnPropertyNames(E), ['length', 'name', 'prototype']))
```

Comment:
Need to remove `includes: [compareArray.js]`

Reply:
Removed both includes.